website/src/contribute/release-guide.md

```diff
@@ -618,6 +647,41 @@ Staging binaries
 
 Verify that files are [present](https://dist.apache.org/repos/dist/dev/beam).
 
+#### Stage SDK images on hub.docker.com
+* Build Python images and push to DockerHub.
```

Comment:
The format seems not correct on staging page: http://apache-beam-website-pull-requests.storage.googleapis.com/9510/contribute/release-guide/index.html#alternative-run-all-steps-manually-1. Can you fix it?

Reply:
It is fixed.